_template/config.yml

```
@@ -1,23 +1,22 @@
 
-# Name of the pack
+# Name of the pack (required)
 name: Template pack
```

For compatibility with the APIs and usage in the CLI, it'd be worthwhile to establish hyphenated lowercase as the format for names
```suggestion
name: template-pack
```